test/components/submission/decrypt.spec.js

```diff
@@ -307,16 +307,6 @@ describe('SubmissionDownload', () => {
       modal.get('a').trigger('click');
       modal.should.alert('info');
     });
-
-    it('does nothing if the link is disabled', () => {
```

Comment:
We test above that the link has the `disabled` class, so we know from the tests of `useDisabled()` that the link will do nothing. I think this test isn't needed anymore (and no longer passes because `useDisabled()` isn't called).